lib/license_finder/decisions.rb

```diff
@@ -210,3 +214,21 @@ def write!(value, file)
     end
   end
 end
+
+module LicenseFinder
+  class RegexParser
+    OPTMAP = {
+        "i" => Regexp::IGNORECASE,
+        "m" => Regexp::MULTILINE,
+        "x" => Regexp::EXTENDED,
+      }
+
+    def self.parse(s)
+      match = s.match(/\/(.*)\/(.*)/) or return
```

Comment:
The regex can be improved to `^\/(.+)\/(.*)$`

The changes are:
 - The whole string must be a regex to work. So test strings like `sometfoobar/apache/i` won't work.
 - We don't want to match `//` so I've changed the first capture group to be `(.+)` instead.

Please add tests for both changes.